### DISCUSSION TITLE
How to display settings changes?

### DISCUSSION THREAD

**Discussion by miklosgergely** [smplayer-dev/smplayer#808] [General]
I used to see feedback information on screen when I changed the speed, or the subtitle delay, but it's not displayed anymore, most likely because I accidentally switched it off. How could I switch it on again? I looked everywhere in the settings, but couldn't find it.

**Comment by smplayer-dev** [comment]
Menu View -> OSD.

**Comment by miklosgergely** [comment]
Thanks! It was Menu View -> OSD -> Volume + Seek, selecting this means that there is still no constant info on screen, just when something is getting set.
